### DISCUSSION TITLE
How to use SSE with Koa?

### DISCUSSION THREAD

**Discussion by TaylorPzreal** [koajs/koa#1774] [Q&A] [answered]
How can I use SSE with Koa? I can't seem to find the @koa/sse NPM package and repository. Has it been deleted? Are there any other options for configuring SSE?

**Comment by sombriks** [comment, marked as answer]
Hi @TaylorPzreal you mean this package isn't available or doesn't work? https://www.npmjs.com/package/koa-sse-stream

can you clarify?
